Makefile

```diff
@@ -1,22 +1,97 @@
-PACKAGES := $(shell glide novendor)
+PKGS := $(shell go list ./... | grep -v go.uber.org/sally/vendor)
+SRCS := $(wildcard *.go)
 
-.PHONY: install
-install:
-	glide --version || go get github.com/Masterminds/glide
+.PHONY: all
+all: test
+
+.PHONY: vendor-update
+vendor-update:
+	rm -rf vendor
+	go get -d -v -t -u -f ./...
+	go get -v github.com/Masterminds/glide
+	glide create
```

Comment:
Do we need `glide create` here? We already have a `glide.yaml`, so I'm not sure what we gain from this.